src/Application.php

```diff
@@ -123,7 +123,7 @@ public function __construct(
      * @throws Exception\BadMethodCallException if the $method is not in $httpRouteMethods.
      * @throws Exception\BadMethodCallException if receiving more or less than 2 arguments.
      */
-    public function __call($method, $args)
+    public function __call($method, array $args)
```

Comment:
Please note this change. If someone override this method need to do the same on his extension, otherwise will get
> Strict Standards: Declaration ...::__call should be compatible with Application::__call($method, array $args)